bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/TargetEnvironment.java

```diff
@@ -68,11 +68,26 @@ public static TargetEnvironment create(Asterism asterism) {
      *
      * @param user list of user targets
      */
-    @Deprecated
-    public static TargetEnvironment create(SPTarget base, GuideEnvironment guide, ImList<UserTarget> user) {
+    public static TargetEnvironment create(final SPTarget base, final GuideEnvironment guide, final ImList<UserTarget> user) {
```

Comment:
I could see no good reason to keep this deprecated, since the other method, `TargetEnvironment.create(SPTarget)` was not, and they seem like good convenience methods.